relevanceai/operations_new/ops_run.py

```diff
@@ -768,25 +768,37 @@ def run(
                 }
             ]
 
+        engine = kwargs.pop("engine", "joseph")
```

Comment:
Can you default this to `jacky` for now?

Reply:
sure